master/networking/vxlan-ipip.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Connecting workloads that you do not control
+---
+
+### Big Picture
+
+Enable inter workload communication across networks that are not workload aware.
+
+### Value
+
+Calico supports encapsulation so you can send traffic between workloads, without requiring the underlying network to be aware of pod IP addresses.
+
+### Features
+
+This how-to guide uses the following features: 
+
+- IP in IP encapsulation
+- VXLAN encapsulation
+
+### Concepts
+
+#### Routing workload IP addresses
+When a network is **aware of workload IP addresses** (either through static routes, BGP route distribution, or another mechanism), it can directly route traffic between workloads. However, not all networks are able to route workload IP addresses. For example, public cloud environments where you don’t own the hardware, AWS across VPC subnet boundaries, and other scenarios where you cannot peer Calico over BGP to the underlay or easily configure static routes. This is why Calico supports encapsulation, so you can send traffic between workloads without requiring the underlying network to be aware of workload IP addresses.
```

Comment:
I think it's best to add newlines after about 120 characters. Otherwise it ends up on a single line which makes editing the content hard in the future.

Reply:
Fixed.